components/admin-panel/Menu.js

```diff
@@ -188,9 +188,10 @@ const Menu = ({ collective, isAccountantOnly }) => {
             collective={collective}
             section={COLLECTIVE_SECTIONS.VIRTUAL_CARDS}
             if={
-              isOneOfTypes(collective, [COLLECTIVE, FUND, EVENT, PROJECT]) &&
-              hasFeature(collective.host, FEATURES.VIRTUAL_CARDS) &&
-              collective.isApproved
+              getCollectiveTypeKey(collective.type) === USER ||
```

Comment:
I don't think it's a good idea do display this in the menu for all users, given that 99% of them don't and will never have a Virtual Card.

Reply:
Yes, I've remove the virtual card section in the user section.